test/utils.el

```diff
@@ -0,0 +1,23 @@
+(require 'dash)
+
+(defmacro codemetrics-test (test-name file major-mode expected-result)
+  "Create a codemetrics test named TEST-NAME using FILE in MAJOR-MODE.
+Expecting result EXPECTED-RESULT."
```

Comment:
We can add `(declare (indent 1))` for better indentation.

Reply:
So that is what those are for! Look way better like you said 😄 (done in latest commits)